rust/src/filetracker.rs

```diff
@@ -15,6 +15,8 @@
  * 02110-1301, USA.
  */
 
+//! Gap handling and Chunk-based file transfer tracker module.
+
 /**
  *  \file
  *  \author Victor Julien <victor@inliniac.net>
```

Comment:
Can we integrate this existing "module" level documentation into this new proper module level documentation?

Reply:
So I should replace the documentation I added with the existing one, yeah?

Reply:
More like merge the 2.

Reply:
So I'll be adding those short explanations present in the existing documentation too?

Reply:
I think so.  If there is documentation there already, it was probably meant to be module level documentation, but not marked up correctly. But the existing documentation may be missing a title like line, which is more or less what you have been adding here.

Reply:
I was thinking leaving it like this would be good, since it already serves as a title-like line, because the existing documentation is quite long and the issue details asked for a short description.

Reply:
Hmm, Ok. One line is better than before.

But idiomatically Rust modules are typically documented like this:

```
//! This is the brief description.
//!
//! Then one or more paragraphs providing more detail about what this module does.
```
not all of our modules have that paragraph, but some like this one do, so would be nice to incorporate it since the content is there already, its just a matter of formatting correctly for rustdoc.

Reply:
Okay then.

I'll do this then.
Thank you for the feedback.